src/explore-education-statistics-frontend/src/modules/find-statistics/FindStatisticsPage.tsx

```diff
@@ -130,7 +130,7 @@ const FindStatisticsPage: NextPage = () => {
         : {
             ...omit(router.query, 'page'),
             [filterType]: nextValue,
-            sortBy: filterType === 'search' ? 'relevance' : sortBy,
+            sortBy: filterType === 'search' ? sortBy : 'newest',
```

Comment:
This is only keeping the user's previous sort order when you search, if you change another filter it resets to 'newest'.